src/app/components/SubjectHeading/BibsList.jsx

```diff
@@ -204,13 +205,14 @@ class BibsList extends React.Component {
     }
     const bibResults = bibsSource === 'discoveryApi' ? results : results.slice(this.firstBib(), this.lastBib());
 
-    const h3Text = `Viewing ${this.firstBib() + 1} - ${this.lastBib()} of ${totalResults || ''} items for Heading "${label}"`;
+    const h3Text = `Viewing ${this.firstBib() + 1} - ${Number.isInteger(totalResults) ? `${this.lastBib()} of ${totalResults} item${totalResults === 1 ? '' : 's'}` : ''}`;
```

Comment:
It still seems like there's the potential to render broken statements like, for example, "'Viewing 1 - " if `totalResults` is non-numeric (or "'Viewing 1 - 0 of 0 items" if `totalResults` is 0), but I'm going to assume that this component is only rendered when `totalResults` is an integer >= 1.